{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<font size=\"4\">\n",
    "Lecture 2 Questions<br>\n",
    "Kohl Kinning<br>\n",
    "7/25/2017 \n",
    "</font>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1. *Draw and label a finished Illumina cDNA library. Define the function of each component.*\n",
    "\n",
    "**Preparing the library**  \n",
    "source: Dr. Wagner, Lecture 2: NGS Platforms\n",
    "<img src=\"./highPass.png\" alt=\"Drawing\" style=\"width: 600px\" align=\"left;\"/>\n",
    "\n",
    "**An example of one item in the Illumina library**   \n",
    "source: https://kscbioinformatics.wordpress.com/category/bioinformatics-resources\n",
    "<img src=\"./libraryfragment.png\" alt=\"Drawing\" style=\"width: 600px\" align=\"left;\"/>\n",
    "\n",
    "2. *What is paired end sequencing? Describe why you would perform paired-end sequencing.*\n",
    "    + The sequencing is duplicated but in the opposite direction. If the first read was from the 5' to the 3' end, the other read will be from the 3' to the 5' end. THis is accomplished using a a primer that attaches to the 3' end. Paired end reads are much easier to align.\n",
    "3. *Why is index hopping a problem? * \n",
    "    1. *draw the proposed mechanism*  \n",
    "    **Proposed mechanism of index hopping**  \n",
    "    source: http://enseqlopedia.com/2016/10/batch-effects-in-scrna-seq-to-e-or-not-to-ercc-spike-in/\n",
    "    <img src=\"./Sinha-et-al-500x383-1.png\" alt=\"Drawing\" style=\"width: 500px\" align=\"left;\"/>  \n",
    "    \n",
    "    2. *describe one solution*\n",
    "        + Here are several tips to mitigate hopping recommended by Illumina:\n",
    "            + Remove free adaptors from library preps\n",
    "            + Store libraries individually at -20°C\n",
    "            + Pool libraries prior to sequencing\n",
    "            + Use unique dual indexing pooling combinations (unique i5 and i7 indexes)\n",
    "        + A solution being implemented by Illumina is to use dual indexes. These will enable researchers to remove unexpected combinations and only focus on the ‘true’ data with correct index combinations.\n",
    "    \n",
    "4. *What are the advantages to Pac Bio? What might be a disadvantage?*\n",
    "    + The PacBio workflow produces very long reads, runs massively parallel and is high throughput, and has a random error mode. This long read sequencer may not be the best for short sequencing needs. \n",
    "5. *Why did Illumina switch to binning quality scores?*\n",
    "    + Illumina bins quality scores to save memory. The ASCII encoding for each read took up too much space.\n",
    "6. *Name two methods of sequencing other than sequencing by synthesis and briefly describe how they work.*\n",
    "    + Roche/454--pyrosequencing\n",
    "        + The process begins with a sample of Double-stranded DNA is fragmented with a restriction enzyme.\n",
    "        + Adapters are attached to the DNA fragments.\n",
    "        + Tiny resin beads are added to the mix.\n",
    "        + The beads contain complementary sequences to the adapters allowing each fragment to bind to a bead.\n",
    "        + When the fragments attach to the beads the DNA denatures.\n",
    "        + The fragments of DNA are then copied numerous times on each bead by PCR.\n",
    "        + The beads are then filtered to remove any that without a fragment or contain more than one DNA fragment.\n",
    "        + The remaining beads are put into wells on a sequencing plate (one bead per well) along with enzyme beads that contain the DNA polymerase and primer needed for the sequencing reaction.\n",
    "        + The polymerase enzyme and primer attach to the DNA fragments on the beads.\n",
    "        + Nucleotide bases are added to the wells one base at a time. Optics record the fluoresence.\n",
    "        + The intensity of the light corresponds to the number of nucleotides of the same type that have been incorporated. \n",
    "        + By plotting this pattern of light intensity on a graph, the sequence of the original piece of DNA can be decoded.\n",
    "    + Ion Torrent\n",
    "        + The process begins with a sample of Double-stranded DNA is fragmented with a restriction enzyme.\n",
    "        + Adapters are attached to the DNA fragments.\n",
    "        + Tiny resin beads are added to the mix.\n",
    "        + The beads contain complementary sequences to the adapters allowing each fragment to bind to a bead.\n",
    "        + When the fragments attach to the beads the DNA denatures.\n",
    "        + The fragments of DNA are then copied numerous times on each bead by PCR.\n",
    "        + The beads are then filtered to remove any that without a fragment or contain more than one DNA fragment.\n",
    "        + The remaining beads are put into wells on a sequencing plate (one bead per well) along with enzyme beads that contain the DNA polymerase and primer needed for the sequencing reaction.\n",
    "        + The polymerase enzyme and primer attach to the DNA fragments on the beads.\n",
    "        + Nucleotide bases are added to the wells one base at a time. When the base is incorporated, an H+ is released. \n",
    "        + The change in pH confirms the incorporation, and corresponds to the number of nucleotides of the same type that have been incorporated.\n",
    "        + By plotting this change in pH, the sequence of the original piece of DNA can be decoded.\n",
    "        \n",
    "7. *Why does Bridge amplification need to occur (on Illumina's flow cell)?*\n",
    "    + Bridge amplifiction is necessary to provide a fluorescent signal strong enough for the optical device to accurately detect. \n",
    "8. *Consult the literature.*\n",
    "    1. *Provide reference one paper each that has used:*  \n",
    "    1. *What was sequencing used to accomplish?*  \n",
    "    1. *Why do you think the authors chose the platform they did?*  \n",
    "        + *Illumina sequencing*  \n",
    "          + 1.Lee, H. et al. Genome Sequence of Sphingomonas sp. Strain PAMC 26621, an Arctic-Lichen-Associated Bacterium Isolated from a Cetraria sp. J Bacteriol 194, 3030 (2012). \n",
    "          + The authors produced a genome of an organism.\n",
    "          + The authors used both the Illumina platform and the Roche platform. The complementary nature of long reads with high error and short reads with low error is likely why the authors chose this platform.\n",
    "\n",
    "        + *Pac Bio sequencing*  \n",
    "          + 1.Smith, H. J., Foreman, C. M. & Ramaraj, T. Draft Genome Sequence of a Metabolically Diverse Antarctic Supraglacial Stream Organism, Polaromonas sp. Strain CG9_12, Determined Using Pacific Biosciences Single-Molecule Real-Time Sequencing Technology. Genome Announc. 2, e01242-14 (2014).\n",
    "          + The authors produced a draft genome of an organism.\n",
    "          + The PacBio platform seem like the top choice for sequencing genomes due to its long read characteristics. Accurate assemblies are better facilitated by long reads.\n",
    "\n",
    "        + *Nanopore sequencing*\n",
    "          + 1.McIntyre, A. B. R. et al. Nanopore sequencing in microgravity. NPJ Microgravity 2, 16035 (2016).  \n",
    "          + The authors were testing whether they could use the MinIon in microgravity to sequence DNA\n",
    "          + The MiniIon has hardly been used. For the most part they have been novelty studies. The MinIon is well suited to serve sequencing requirements in astrobiological situations.  \n",
    "\n",
    "        + *Ion Torrent sequencing*  \n",
    "            + 1.Semenova, T. A. et al. Compositional and functional shifts in arctic fungal communities in response to experimentally increased snow depth. Soil Biology and Biochemistry 100, 201–209 (2016).  \n",
    "            + The authors use DNA metabarcoding to study the effects of long-term experimental manipulations of snow depth on soil fungal communities in dry heath and moist tussock tundra in Arctic Alaska.\n",
    "            + The speed of the process may have been another reason they chose this platform. The authors were sequencing soil communites and so desired a high throuput platform. The Ion Torrent filled this need. They chose the Ion 318 Chip to allow for highest possible sequencing coverage.\n",
    "            \n",
    " \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.1"
  },
  "toc": {
   "colors": {
    "hover_highlight": "#DAA520",
    "navigate_num": "#000000",
    "navigate_text": "#333333",
    "running_highlight": "#FF0000",
    "selected_highlight": "#FFD700",
    "sidebar_border": "#EEEEEE",
    "wrapper_background": "#FFFFFF"
   },
   "moveMenuLeft": true,
   "nav_menu": {
    "height": "12px",
    "width": "252px"
   },
   "navigate_menu": true,
   "number_sections": true,
   "sideBar": true,
   "threshold": "3",
   "toc_cell": false,
   "toc_section_display": "block",
   "toc_window_display": false,
   "widenNotebook": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
